lib/processing.py

```diff
@@ -0,0 +1,19 @@
+# libraries
+#import numpy as np
+from sensors import Sensor
+import yaml
+import sys
+import os
+# disaster_scenes
+config_file = sys.argv[1]
+with open(config_file,'r') as file_stream:
+    try:
+        config = yaml.load(file_stream)
+    except yaml.YAMLError as exception:
+        print(exception)
```

Comment:
Why just print the exception, instead of killing the whole process by just letting it be raised?